src/values/NativeFunctionValue.js

```diff
@@ -83,6 +83,10 @@ export default class NativeFunctionValue extends FunctionValue {
     }
   }
 
+  hasDefaultLength(): boolean {
+    return true;
+  }
+
```

Comment:
What is the meaning of hasDefaultLength() function? Maybe we do not need to override it in NativeFunctionValue class?